integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/model/DefaultDecimalScaleMappingIT.java

```diff
@@ -37,8 +37,8 @@ public void mapping() {
 		backendMock.expectSchema( INDEX_NAME, b -> b
 				.field( "scaled", BigDecimal.class, f -> f.defaultDecimalScale( 7 ) )
 				.field( "unscaled", BigDecimal.class, f -> f.defaultDecimalScale( 0 ) )
-				.field( "defaultScaled", BigDecimal.class, f -> f.defaultDecimalScale( 2 ) )
-				.field( "hsearch", BigDecimal.class, f -> f.defaultDecimalScale( 2 ).decimalScale( 7 ) )
+				.field( "defaultScaled", BigDecimal.class, f -> { } )
```

Comment:
This is not really a fix. With the stub backend, it will work, granted. But with real backends, decimal fields *require* a decimal scale, so this will lead to a mapping error, whereas in ORM 5 it would have worked and defaulted to 2.

I suppose ORM 6 still provides a default scale, but handles it differently. If so, we need to get this scale from Hibernate ORM metadata and communicate it to the backend somehow, similarly to what we do in `org.hibernate.search.mapper.orm.mapping.impl.HibernateOrmMappingPropertiesMetadataContributor#collectScale`.

If that's not possible, then we need to either decide on a backwards-compatible default ourselves, or document the limitation somewhere.

In any case, just changing the test to accept incorrect behavior doesn't seem the right call. Let's keep the test failing for now, to remind us there is something to fix here?

Reply:
Sure, makes sense!